tools/lib/ignoredPaths.js

```diff
@@ -3,7 +3,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 module.exports = [
-  'pre-comit',
+  'pre-commit',
```

Comment:
I've been not sure if this was a typo: https://github.com/brave/browser-laptop/pull/9690#discussion_r123872847

Reply:
cc @NejcZdovc on whether this is a typo

Reply:
These are sub-dirs of node_modules mostly so yes it's a typo. Thx.